Dangerfile

```diff
@@ -1,7 +1,7 @@
 # Warn if a pull request is too big
 PR_SIZE = {
-  RECOMMENDED_MAXIMUM: 250,
-  ABSOLUTE_MAXIMUM:    1000,
+  RECOMMENDED_MAXIMUM: 200,
+  ABSOLUTE_MAXIMUM:    800,
```

Comment:
I think the RECOMMENDED_MAXIMUM of 200 makes sense, but i think without whitespace counting we should push the ABSOLUTE_MAXIMUM even lower.  I don't think I've seen many PRs approach the 1k that didn't need to be broken up, and without whitespace changes I bet it could be a lower maximum.  But I think I guess I also think we'll get data faster if we set it too low and have to increase it.
I would put it at 500, but I'm going to solicit some more feedback.

Reply:
I think 500 is good middle ground but without taking a look at historical danger-warnings it's sort of guess-n-check anyhow.